>>> 'Pkg.add("GLM")' log
INFO: Cloning cache of GLM from git://github.com/JuliaStats/GLM.jl.git
INFO: Installing ArrayViews v0.4.8
INFO: Installing Distributions v0.5.10
INFO: Installing GLM v0.4.2
INFO: Installing NumericExtensions v0.6.2
INFO: Installing NumericFuns v0.2.3
INFO: Installing PDMats v0.3.1
INFO: Installing StatsBase v0.6.10
INFO: Package database updated

>>> 'using GLM' log
Warning: could not import Base.add! into NumericExtensions
Julia Version 0.3.1
Commit c03f413* (2014-09-21 21:30 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3

>>> test log
Warning: could not import Base.add! into NumericExtensions
Running tests:
 * test/glmFit.jl ...

ERROR: could not open file /home/idunning/pkgtest/.julia/v0.3/GLM/test/glmFit.jl
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in anonymous at no file:10
 in include at ./boot.jl:245
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354
 in _start_3B_1712 at /home/idunning/julia03/usr/bin/../lib/julia/sys.so
while loading /home/idunning/pkgtest/.julia/v0.3/GLM/runtests.jl, in expression starting on line 7

>>> end of log
